security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Restricting Fiddler
```

Comment:
I suggest we use the name of the feature as a headline as well: Managed App Configuration